ovn-kubernetes/ovn-kubernetes
# Increase scalability of secondary network controllers by multiplexing informers.

## 📑 Description
Instead of using a single (federated) event handler for the shared kubernetes index informer create a pool of handlers.  Ensures that all secondary network controllers register their object event handlers with the same entry in the pool.  This allows load sharing between secondary network controllers.

## Additional Information for reviewers
The change is mostly limited to the `factory` package but requires shallow cloning of the watch factory when secondary network controllers are created.

## ✅ Checks


## How to verify it
All existing tests must pass.  The change should be transparent from a functionality point of view.
